app/src/main/java/com/sopt/peekabookaos/presentation/userInput/UserInputViewModel.kt

```diff
@@ -127,6 +127,13 @@ class UserInputViewModel @Inject constructor(
         _isDuplicateButton.value = state
     }
 
+    fun removeCheckMessage() {
+        if (_isNicknameMessage.value == false) {
+            _isCheckMessage.value = false
+            updateCheckMessage(false)
+        }
+    }
```

Comment:
```suggestion
    fun removeCheckMessage() {
        if (!_isNicknameMessage.value) {
            _isCheckMessage.value = false
            updateCheckMessage(false)
        }
    }
```
사소한건데 이런식으로도 쓸 수 있어!